doc/cug.tex

```diff
@@ -2822,9 +2822,13 @@ \subsubsection{Graph Types}
             graph = "delayed_cold_corge => corge"
 
         # Repeat once at the final cycle point
-        [[[ R1/P0Y ]]]
+        [[[ R1/$ ]]]
             graph = "end_garply"
 
+        # Repeat once three days before the final cycle point
```

Comment:
This is consistent with our existing documentation, but I wonder if we should change "Repeat once" to "Run once" - the former is nonsensical if interpreted correctly (to repeat something once really suggest to doing it twice)??